tests/jballerina-unit-test/src/test/resources/test-src/expressions/literals/numeric_literal_assignment.bal

```diff
@@ -123,9 +123,9 @@ function testFloatLiteralAsDecimalInUnion() returns boolean {
 }
 
 function testFloatLiteralAsDecimalInUnion_2() returns boolean {
-    Bar|decimal y = 123.0;
-    decimal dec = 123.0;
-    return y is decimal && dec == y;
+    Bar|decimal y = 12.0;
+    float dec = 12.0;
```

Comment:
The name of the function is `testFloatLiteralAsDecimalInUnion_2`. Does this logic still match that?